editor/src/core/webpack-loaders/file-loader.ts

```diff
@@ -0,0 +1,17 @@
+import { ProjectFile } from '../shared/project-file-types'
+import { LoadModule, MatchFile, ModuleLoader } from './loader-types'
+
+const matchFile: MatchFile = (filename: string) => {
+  return ['.avif', '.bmp', '.gif', '.jpg', '.jpeg', '.png'].some((extension) =>
+    filename.endsWith(extension),
+  )
+}
+
+const loadModule: LoadModule = (filename: string, _: ProjectFile) => {
```

Comment:
you can omit `_: ProjectFile` and it will still be correctly typed – unless you wanted to leave this here as an example